From: Gordon Ball <gordon@chronitis.net>
Date: Tue, 6 Feb 2018 12:17:37 +0000
Subject: Don't try and patch bootstrap less

setupbase.py tries to patch out one line from bootstrap's less, to
improve printing. In this case we're not using a private copy of this
file, so we can't modify the system copy.

TODO: Find an alternative way of implementing this fix.
---
 setupbase.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/setupbase.py b/setupbase.py
index 5839d87..267e013 100644
--- a/setupbase.py
+++ b/setupbase.py
@@ -478,8 +478,6 @@ class CompileCSS(Command):
         env = os.environ.copy()
         env['PATH'] = npm_path
 
-        patch_out_bootstrap_bw_print()
-        
         for src, dst in zip(self.sources, self.targets):
             try:
                 run(['lessc',
